WordPress/Classes/System/Constants.m

```diff
@@ -14,7 +14,7 @@
 NSString *const WPAutomatticTermsOfServiceURL                       = @"https://wordpress.com/tos/";
 NSString *const WPAutomatticPrivacyURL                              = @"https://automattic.com/privacy/";
 NSString *const WPAutomatticCookiesURL                              = @"https://automattic.com/cookies/";
-NSString *const WPAutomatticAppsBlogURL                             = @"https://apps.wordpress.com/blog/";
+NSString *const WPAutomatticAppsBlogURL                             = @"https://http://blog.wordpress.com";
```

Comment:
It looks like we have two sets of protocols here – probably best to just go with `https://blog.wordpress.com`.